### ISSUE TITLE
Proposal to Store WASM code should reference the wasm code being uploaded

### ISSUE THREAD

**Issue by hard-nett** [CosmWasm/wasmd#1488]
Would be nice to instead of need to manually copy & paste the wasm_byte_code into the draft proposal, to reference the wasm file location in the workflow of drafting a proposal.

**Comment by alpe** [comment]
🤔 Can you help me with some examples to understand your use case better? 
We have `wasmd tx wasm submit-proposal` for v1 gov proposals. The `wasmd tx gov draft-proposal` from the sdk does not work nicely with wasm proposal types. There is no way to integrate and make this more convenient. The SDK team recommended to provide our own v1 gov command instead.

**Comment by hard-nett** [comment]
@alpe no problem! On a permissioned wasm chain, after running `wasmd tx gov draft-proposal`, you end up with the `draft-proposal` file that includes `null` in the `wasm_byte_code` part of the drafted proposal. 

I was envisioning during calling `wasmd tx gov draft-proposal` to include referencing the .wasm file locally to include it into the proposal file.

**Comment by hard-nett** [comment]
ah ok, took a look at the gov module and found [here](https://github.com/cosmos/cosmos-sdk/blob/v0.47.3/x/gov/client/cli/prompt.go#L236) where the proposal type gets messy with wasm proposal type.

**Comment by alpe** [comment]
Thanks for the additional details! I had run into the same issues before. 
We discussed different solutions to the problem in https://github.com/CosmWasm/wasmd/issues/1301 . As there is no way to integrate with the reflection based approach in the SDK, we had added the `wasmd tx wasm submit-proposal` command.
This should work for you as in previous wasmd versions but is limited to only 1 message in your proposal.
I hope that gives you more context. There is nothing we can do now without forking the SDK command.
